client/workitemtype.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
+	"net/http"
+	"net/url"
+)
+
+// ShowWorkitemtypePath computes a request path to the show action of workitemtype.
+func ShowWorkitemtypePath(id string) string {
```

Comment:
Renaming the resource to e.g. "workItemType" leads to duplicate types in the generated code. Leaving it as it is for now.